<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileInformationsDialog</class>
 <widget class="QDialog" name="FileInformationsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>260</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Informations sur le fichier audio</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons/icons.qrc">
    <normaloff>:/sound/audio.png</normaloff>:/sound/audio.png</iconset>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="file_info_layout">
     <item>
      <layout class="QHBoxLayout" name="file_info_layout_path">
       <item>
        <widget class="QLabel" name="file_info_label_path">
         <property name="minimumSize">
          <size>
           <width>126</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>126</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Chemin</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="file_info_path">
         <property name="toolTip">
          <string>Chemin vers le fichier</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="file_info_layout_size">
       <item>
        <widget class="QLabel" name="file_info_label_size">
         <property name="minimumSize">
          <size>
           <width>126</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>126</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Taille du fichier</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="file_info_size">
         <property name="toolTip">
          <string>Taille du fichier</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="file_info_layout_codec">
       <item>
        <widget class="QLabel" name="file_info_label_codec">
         <property name="minimumSize">
          <size>
           <width>126</width>
           <height>22</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>126</width>
           <height>22</height>
          </size>
         </property>
         <property name="text">
          <string>Codec audio</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="file_info_codec">
         <property name="toolTip">
          <string>Codec audio</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="file_info_layout_bitrate">
       <item>
        <widget class="QLabel" name="file_info_label_bitrate">
         <property name="minimumSize">
          <size>
           <width>126</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>126</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Bitrate</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="file_info_bitrate">
         <property name="toolTip">
          <string>Qualité audio</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="file_info_layout_sample_rate">
       <item>
        <widget class="QLabel" name="file_info_label_sample_rate">
         <property name="minimumSize">
          <size>
           <width>126</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>126</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Freq échantillonage</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="file_info_sample_rate">
         <property name="toolTip">
          <string>Fréquence d'échantillonage</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="file_info_layout_nb_channels">
       <item>
        <widget class="QLabel" name="file_info_label_nb_channels">
         <property name="minimumSize">
          <size>
           <width>126</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>126</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>Nombre de voies</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="file_info_nb_channels">
         <property name="toolTip">
          <string>Nombre de canaux audio</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="action_close">
       <property name="toolTip">
        <string>Fermer</string>
       </property>
       <property name="text">
        <string>Fermer</string>
       </property>
       <property name="shortcut">
        <string>Esc, Return</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_close</sender>
   <signal>clicked()</signal>
   <receiver>FileInformationsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
